compiler/onecc-docker/onecc-docker

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import subprocess
+import json
+import requests
+
+
+def _run(cmd, is_shell=False):
+    result = subprocess.Popen(
+        cmd, shell=is_shell, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+    stdout, stderr = result.communicate()
+
+    if stderr:
+        print(stderr.decode('utf-8'))
+        exit(1)
+    else:
+        return stdout
+
+
+def _image_exists(name):
+    cmd = ['docker', 'images', '-q', name]
+    lines = _run(cmd).splitlines()
+    return lines
+
+
+def main():
+    Dockerfile_path = "."
+
+    TAG_URL = "https://api.github.com/repos/Samsung/ONE/releases/latest"
+    try:
+        response = requests.get(TAG_URL)
+        response.raise_for_status()
+    except requests.exceptions.RequestException as e:
+        raise SystemExit(e)
+
+    versions_str = response.content
+    versions_json = json.loads(versions_str)
+    recent_version = versions_json["tag_name"]
+
+    image_name = f"onecc:{recent_version}"
+    build_arg = f"VERSION={recent_version}"
+
+    if not _image_exists(image_name):
+        build_cmd = [
+            "docker", "build", "-t", image_name, "--build-arg", build_arg, Dockerfile_path
+        ]
+        print('build Docker image ...')
+        _run(build_cmd)
+        print('Dockerfile successfully built.')
+
+    contianer_name = f"onecc_{recent_version.replace('.','_')}"
+    user_cmd = ''.join(sys.argv[1:])
+    user_groups = _run("id --groups", True).decode('utf-8').replace("\n", "").split(' ')
+    docker_groups = ' --group-add '.join(user_groups)
+
+    run_cmd = [
+        "docker", "run", "--rm", "-u", "$(id -u):$(id -g)", "--group-add", docker_groups,
+        "--name", contianer_name, "-v", "${HOME}:${HOME}", "-e", "HOME=${HOME}", "-w",
+        "${PWD}", image_name, user_cmd
+    ]
+
+    cmd = ' '.join(run_cmd)
+    output = _run(cmd, is_shell=True)
+    print(output.decode('utf-8'), end='')
```

Comment:
Q) I'm not that good at python but is it ok not to wait for `subprocess.Popen` to end?
or what happens if return process some problem?
Adding more fixes may take some time so you may add some `# TODO ...` comments to do in later time so to land this PR.

Reply:
> I'm not that good at python but is it ok not to wait for subprocess.Popen to end?
or what happens if the child process some problem?

I think I can use `subprocess.communicate()` to wait until the child process is over...
Is your concern about timeout? 

It's not perfect, but if there's a problem, I printed an error and set it to end the process.

> Adding more fixes may take some time so you may add some # TODO ... 

Should I add TODO in the PR?

@jyoungyun , can you give us any suggestions on this?

Reply:
As I know, this command waits for command to complete. So it's okay. :)